_posts/2021-03-10-Bootable-jar-jkube-clustering-dns-ping-openshift.adoc

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+date:   2021-03-10
+tags: wildfly galleon bootable jar bootable-jar clustering jkube dns-ping
+author: yborgess
+---
+
+= Configuring DNS ping on a WildFly Bootable JAR cluster application
+
+[NOTE]
+====
+This post is a continuation of https://www.wildfly.org/news/2021/02/01/Bootable-jar-jkube-clustering-openshift[WildFly Bootable JAR cluster application with JKube openshift-maven-plugin] blog post, it assumes you are familiar with the https://github.com/yersan/wildfly-clustering-demo.git[demo application] we used for that.
+====
+
+== Introduction
+
+We recently described in this https://www.wildfly.org/news/2021/02/01/Bootable-jar-jkube-clustering-openshift[post] details about how to deploy and configure a WildFly Bootable JAR cluster application on OpenShift. By default, a Bootable JAR cluster application uses the http://www.jgroups.org/manual5/index.html#_kube_ping[KUBE_PING] protocol for clustering discovery mechanism. This protocol is based on the ability to query the Kubernetes API to determine which pods match certain criteria and then create a JGroups cluster based on this information. This protocol could not be a viable or an ideal solution for your scenario, for example, if you cannot grant a specific permission on your Kubernetes cluster to use KUBE_PING, or if your requirement is to make your application agnostic to the Kubernetes API.
```

Comment:
s/for clustering discovery/for its clustering discovery/g

s/could not/may not/g

s/scenario, for example/scenario; for example/g